pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -873,12 +923,17 @@ public void UpdateUI(bool selectionChanged = true)
                                 var imText = string.Empty;
                                 var ccs = libraryChromGroup?.CCS ?? dt.CollisionalCrossSectionSqA;
                                 if (ccs.HasValue)
+                                {
                                     ccsText = Resources.ViewLibraryDlg_UpdateUI_CCS__ + string.Format(@"{0:F2}", ccs.Value);
+                                    _currentProperties.CCS = string.Format(@"{0:F2}", ccs.Value);
+                                }
                                 if (dt.HasIonMobilityValue)
                                     imText = Resources.ViewLibraryDlg_UpdateUI_IM__ + string.Format(@"{0:F2} {1}", dt.IonMobility.Mobility, dt.IonMobility.UnitsString);
                                 if ((dt.HighEnergyIonMobilityValueOffset??0) != 0) // Show the high energy value (as in Waters MSe) if different
                                     imText += String.Format(@"({0:F2})", dt.HighEnergyIonMobilityValueOffset);
                                 labelRT.Text = TextUtil.TextSeparate(@"  ", labelRT.Text, ccsText, imText);
+                                
+                                _currentProperties.IonMobility = string.Format(@"{0:F2} {1}", dt.IonMobility.Mobility, dt.IonMobility.UnitsString);

```

Comment:
This looks identical to the above:
if (dt.HasIonMobilityValue)
    imText = Resources.ViewLibraryDlg_UpdateUI_IM__ + string.Format(@"{0:F2} {1}", dt.IonMobility.Mobility, dt.IonMobility.UnitsString);
    
Seems like this should go into that if statement block, and the string.Format() should only be done once.